noxfile.py

```diff
@@ -14,9 +14,6 @@
     "src/urllib3/_collections.py",
     "src/urllib3/fields.py",
     "src/urllib3/filepost.py",
-    "src/urllib3/packages/__init__.py",
-    "src/urllib3/packages/ssl_match_hostname/__init__.py",
-    "src/urllib3/packages/ssl_match_hostname/_implementation.py",
```

Comment:
@pquentin Should we keep the new `src/util/ssl_match_hostname.py` in this list?

Reply:
We should yes!

Reply:
Ok, I will add type-hints to `src/util/ssl_match_hostname.py` then I will add the file to the list.